### DISCUSSION TITLE
Column level permission

### DISCUSSION THREAD

**Discussion by FarhanAliRaza** [pocketbase/pocketbase#4058] [Q&A]
I have an application where I give users daily rewards, which increases the limit that number in the user collection.. So my problem is that the user should be able to update their profile but not that limit property because that is only increased if prechecks like daily login are met.

**Comment by ganigeorgiev** [comment]
If I'm understanding your use case, one possible solution could be to specify the [`:isset`](https://pocketbase.io/docs/api-rules-and-filters/#isset-modifier) modifier for your field (for both create and update API rules):

```js
// disallow user from submitting yourField
@request.data.yourField:isset = false
```

Alternatively, you can move the private/protected fields to a helper collection with its own API rules and reference back to the main one with a `relation` field.

**Reply by FarhanAliRaza** [reply]
if I use a helper collection will have to use the admin instance of Pocketbase to change programmatically?

**Reply by ganigeorgiev** [reply]
It depends on your desired behavior and who needs to have write access to the private fields.
You can leave the create/update of the helper collection "Admin only" and create/update its records programmatically using [Go](https://pocketbase.io/docs/go-records/) or [JS](https://pocketbase.io/docs/js-records/).
